docs/content/guides/non-argument-deps.mdx

```diff
@@ -0,0 +1,136 @@
+
+However, often we don't want to build or load our assets as Python objects in memory. For example, we might compute an asset by copying a file, by issuing a `create table` statement to a database, or by executing a command-line utility.
+
+In those situations, we can define assets whose functions don't have return values or arguments. And we can mix those assets with assets that do have return values or arguments.
+
+---
+
+## Bringing in more stopwords
+
+In the [tutorial](/tutorial), we defined an asset that contains a list of stopwords to filter out common words or prepositions from our word cloud. The English language has a lot of stopwords, so rather than hardcoding them into your asset, you'll bring them in from a separate data source.
+
+You'll augment your asset graph with a new asset that contains the stopwords from a zip file on the internet and create an asset from those stopwords. These stopwords will replace the hardcoded stopwords in your `topstories_word_cloud` asset. By the end of this guide, your asset graph will have two new assets:
+
+- `stopwords_zip` - A zip file on our local filesystem, which contains the stopwords.
+- `stopwords_csv` - A list of stopwords, which contains the unzipped contents of `stopwords_zip`.
+
+### A zip file of stopwords from the internet
+
+Let's start with `stopwords_zip`. We'll add this to the same `assets.py` you wrote in the [tutorial](/tutorial):
+
+```python
+import urllib.request # Addition: new import at the top of `assets.py`
+
+@asset
+def stopwords_zip() -> None:
+    urllib.request.urlretrieve(
+        "https://docs.dagster.io/assets/stopwords.zip",
+        "stopwords.zip",
+    )
+```
+
+The implementation downloads a zip file from the internet to our local filesystem. It looks a lot like the assets we defined earlier in this tutorial, with two differences:
+
+- **It doesn't have a return statement.** Instead of returning an object and letting the I/O manager write it to storage, the function handles writing to storage itself.
+- **It has a `None` annotation for its return type.** This helps Dagster understand what we already know by looking at it - that the function never returns any values.
+
+### An unzipped CSV of stopwords
+
+Let's define an asset that contains the unzipped contents of our zip file:
+
+```python
+import zipfile
+
+@asset(non_argument_deps={"stopwords_zip"})
+def stopwords_csv() -> None:
+    with zipfile.ZipFile("stopwords.zip", "r") as zip_ref:
+        zip_ref.extractall(".")
+```
+
+The implementation extracts our zip file to a CSV file in the same directory. It looks a lot like the assets we defined earlier in this tutorial, with one main difference.
+
+In the assets we've seen so far, we indicate the asset dependencies by including function arguments with the name of the upstream assets. Dagster then invokes an I/O manager to load the asset as a Python object and supply it to our function.
```

Comment:
> In the assets we've seen so far,

❓ 🔧  So far in this guide, we've seen two assets. Does this sentence apply fully to those assets? In this case, we've only seen `non_argument_deps`, which I don't think is what this sentence is talking about